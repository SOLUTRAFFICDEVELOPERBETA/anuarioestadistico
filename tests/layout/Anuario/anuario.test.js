import React from 'react';
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import AnuarioEstadistico from '../../../Layout/Anuario';


describe('Pruebas en AnuarioEstadistico', () => {
    const children = (<p>Hijo del componente</p>)
    const wrapper = shallow(
        <AnuarioEstadistico children={children} />
    )
    test('debe de mostrar el componente correctamente', () => {
        expect(wrapper).toMatchSnapshot();
    })

    test('debe de mostrar la estructura enviada', () => {
        const p = wrapper.find('p');
        expect(p.text().trim()).toBe('Hijo del componente');
    })
})