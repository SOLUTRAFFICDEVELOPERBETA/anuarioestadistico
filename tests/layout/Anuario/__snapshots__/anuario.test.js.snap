// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en AnuarioEstadistico debe de mostrar el componente correctamente 1`] = `
<div>
  <div>
    <p>
      Hijo del componente
    </p>
  </div>
</div>
`;
